minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterControl.java

```diff
@@ -116,6 +116,8 @@ public void adminStopAll() throws IOException {
     if (p.exitValue() != 0) {
       throw new IOException("Failed to run `accumulo admin stopAll`");
     }
+    stopAllServers(ServerType.COMPACTOR);
+    stopAllServers(ServerType.SCAN_SERVER);
```

Comment:
This change could be made to 2.1 or 3.x versions, but it would have no affect. Only in `elasticity` are the COMPACTOR and SCAN_SERVER server types started by default in MiniAccumuloCluster.